### PR TITLE
CH-15181 Add `affiliateId` header to checkout & basket requests

### PR DESCRIPTION
**Pull request by MikePolyakovsky**

### CODE REVIEW

File: src/basket-service/services/__tests__/api-provider.spec.ts [hunk 1]
```diff
@@ -25,10 +25,15 @@ describe('Basket API', () => {
     sourceVersion: 'Source version',
   };
   const basketApi = getBasketServiceApi(Environment.Dev, null, sourceInformation);
+  const testChannelId = 'test-qa-encoretickets';
   const additionalHeaders = {
     'x-ttg-client': 'Source name | View name using JS SDK',
     'x-ttg-client-version': 'Source version',
   };
+  const headersWithAffiliate = {
+    ...{affiliateId: 'encoretickets'},
```
Comment (PaulRumyantsevItr) [inline src/basket-service/services/__tests__/api-provider.spec.ts:34]: Spaces missing inside brackets
`...{ affiliateId: 'encoretickets' },`

File: src/checkout-service/services/__tests__/api-provider.spec.ts [hunk 2]
```diff
@@ -27,6 +27,10 @@ describe('Checkout Api', () => {
     'x-ttg-client': 'Source name | View name using JS SDK',
     'x-ttg-client-version': 'Source version',
   };
+  const headersWithAffiliate = {
+    ...{affiliateId: 'encoretickets'},
```
Comment (PaulRumyantsevItr) [inline src/checkout-service/services/__tests__/api-provider.spec.ts:31]: Same as above

**Review by PaulRumyantsevItr — CHANGES_REQUESTED** [review]
We should also mention all available parameters in the documentation (README file)

**Comment by MikePolyakovsky** [comment]
> We should also mention all available parameters in the documentation (README file)

fixed

File: src/basket-service/services/__tests__/api-provider.spec.ts (hunk 1 above)
Comment (MikePolyakovsky) [reply src/basket-service/services/__tests__/api-provider.spec.ts]: fixed

File: src/checkout-service/services/__tests__/api-provider.spec.ts (hunk 2 above)
Comment (MikePolyakovsky) [reply src/checkout-service/services/__tests__/api-provider.spec.ts]: fixed

**Comment by PaulRumyantsevItr** [comment]
Please, update the version in package.json and package-lock.json

**Comment by MikePolyakovsky** [comment]
> We should also mention all available parameters in the documentation (README file)

fixed (1.13.0)

**Review by PaulRumyantsevItr — APPROVED** [review]

**Comment by dpiletski** [comment]
Attaching to the cloubhouse story [15181]

**Pull request merged**